http_cache/esi.rst

```diff
@@ -156,9 +156,9 @@ used ``render()``.
 
 .. note::
 
-    Symfony detects if a gateway cache supports ESI via another Akamai
-    specification that is supported out of the box by the Symfony reverse
-    proxy.
+    Symfony considers that a gateway cache supports ESI if its responses include
```

Comment:
it is in the request, not in the response

Reply:
Fixed! Thanks!

Reply:
Good catch! @OskarStark could you please update this in a direct merge in upstream? No need to make a PR for this. Thanks!

Reply:
Already done :-)